src/main/kotlin/com/vanniktech/dependency/graph/generator/DependencyGraphGenerator.kt

```diff
@@ -77,24 +77,25 @@ internal class DependencyGraphGenerator(
     val identifier = (dependency.moduleGroup + dependency.moduleName).dotIdentifier
     val pair = parentIdentifier to identifier
 
-    if (generator.include.invoke(dependency) && !addedConnections.contains(pair)) {
-      addedConnections.add(pair)
+    if (pair in addedConnections) return
```

Comment:
Is there a reason you rewrote this? I personally liked the previous if statement a lot more

Reply:
Nah then let's keep and next time I'll look at each commit of yours. I only looked at the overall changes